scripts/release_scripts/increment_version.py

```
@@ -0,0 +1,144 @@
+def get_git_commits_since_last_version(last_version: str) -> str:
+    """Get all git commits since the last version."""
+    result = subprocess.run(
+        ["git", "log", f"{last_version}..HEAD", "--pretty=format:%s (%h)"],
+        capture_output=True,
+        text=True,
+    )
+    if result.returncode != 0:
+        raise Exception(f"Error running git log: {result.stderr}")
+    return result.stdout.strip()
+
+import re
```

Ruff

Redefinition of unused `re` from line 2

Remove definition: `re`

(F811)